plugin/src/main/java/com/craftmend/openaudiomc/OpenAudioMc.java

```diff
@@ -89,6 +91,7 @@
             .registerTypeAdapter(AbstractPacketPayload.class, new AbstractPacketAdapter())
             .registerTypeAdapter(ShowRunnable.class, new RunnableTypeAdapter())
             .registerTypeAdapter(OARedisPacket.class, new RedisTypeAdapter())
+            .registerTypeAdapter(ExecuteBulkCommandsPacket.class, new RedisTypeAdapter())
```

Comment:
so i think `RedisTypeAdapter` should be registered for `ExecuteBulkCommandsPacket` explicitely (and also probably for `ExecuteCommandPacket`) in order to actually be used when serializing/deserializing